litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/styles.ts

```diff
@@ -115,8 +117,9 @@ const useStyles = makeStyles((theme: Theme) => ({
     '& p': {
       paddingRight: theme.spacing(2),
     },
-    marginRight: theme.spacing(5),
-    paddingRight: theme.spacing(5),
+    padding: 0,
+    marginRight: theme.spacing(5.75),
+    paddingRight: theme.spacing(5.75),
```

Comment:
use one padding style specify all in one line

Reply:
If the requirement is setting all padding to 0 and changing only one...this is fine. @ishangupta-ds 
but is the `padding: 0` line required? @whitetiger1399